### PR TITLE
web: more specific info when showing nudge - whether user opted in/out, http response body if error; finalize copy [ch2553]

### PR DESCRIPTION
**Pull request by maiamcc**
Hello @jazzdan, @landism,

final final copy tbd (db is working on it) but wanted to get this out for review

### CODE REVIEW

File: web/src/AnalyticsNudge.test.tsx [hunk 1]
```diff
@@ -22,18 +22,30 @@ it("shows request-in-progress message", () => {
...
+
+  expect(component).toMatchSnapshot()
+})
+
+it("shows failure message with response body", () => {
+  const component = shallow(<AnalyticsNudge needsNudge={false} />)
+
+  component.setState({
+    requestMade: true,
+    responseCode: 418,
```
Comment (jazzdan) [inline web/src/AnalyticsNudge.test.tsx]: Is this [a reference](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/418)? :P

File: web/src/AnalyticsNudge.tsx [hunk 2]
```diff
@@ -2,14 +2,49 @@ import React, { Component } from "react"
 import "./AnalyticsNudge.scss"
 
 const nudgeTimeoutMs = 3000 // 3 seconds
+let nudgeElem = (): JSX.Element => {
```
Comment (jazzdan) [inline web/src/AnalyticsNudge.tsx]: This could be a `const` too, FWIW.

Comment (maiamcc) [reply web/src/AnalyticsNudge.tsx]: 🤯

File: web/src/AnalyticsNudge.test.tsx (hunk 1 above)
Comment (maiamcc) [reply web/src/AnalyticsNudge.test.tsx]: but of course!

**Pull request merged**